components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
Safest to throw the error instead of acting on the wrong object? Especially for admin stuff.